test/api/peer/peer.transactions.main.ws.js

```diff
@@ -1,89 +1,60 @@
 'use strict';
 
 var crypto = require('crypto');
-var node = require('./../node.js');
+var node = require('../../node.js');
+var http = require('../../common/httpCommunication.js');
+var ws = require('../../common/wsCommunication.js');
 
 var genesisblock = require('../../genesisBlock.json');
 
 function postTransaction (transaction, done) {
-	node.post('/peer/transactions', {
+	ws.call('postTransactions', {
 		transaction: transaction
-	}, done);
+	}, done, true);
 }
 
 function getAddress (address, done) {
-	node.get('/api/accounts?address=' + address, done);
+	http.get('/api/accounts?address=' + address, done);
 }
 
-describe('GET /peer/transactions', function () {
-
-	it('using incorrect nethash in headers should fail', function (done) {
-		node.get('/peer/transactions')
-			.set('nethash', 'incorrect')
-			.end(function (err, res) {
-				node.debug('> Response:'.grey, JSON.stringify(res.body));
-				node.expect(res.body).to.have.property('success').to.be.not.ok;
-				node.expect(res.body.expected).to.equal(node.config.nethash);
-				done();
-			});
-	});
-
-	it('using incompatible version in headers should fail', function (done) {
-		node.get('/peer/transactions')
-			.set('version', '0.1.0a')
-			.end(function (err, res) {
-				node.debug('> Response:'.grey, JSON.stringify(res.body));
-				node.expect(res.body).to.have.property('success').to.be.not.ok;
-				node.expect(res.body).to.have.property('message').to.eql('Request is made from incompatible version');
-				node.expect(res.body).to.have.property('expected').to.eql('0.0.0a');
-				node.expect(res.body).to.have.property('received').to.eql('0.1.0a');
-				done();
-			});
-	});
+describe('getTransactions', function () {
 
 	it('using valid headers should be ok', function (done) {
-		node.get('/peer/transactions')
-			.end(function (err, res) {
-				node.expect(res.body).to.have.property('success').to.be.ok;
-				node.expect(res.body).to.have.property('transactions').to.be.an('array');
-				done();
-			});
+		ws.call('getTransactions', function (err, res) {
+			node.expect(res).to.have.property('success').to.be.ok;
+			node.expect(res).to.have.property('transactions').to.be.an('array');
+			done();
+		});
 	});
-});
 
-describe('POST /peer/transactions', function () {
+	before(function (done) {
+		var randomAccount = node.randomAccount();
+		var transaction = node.lisk.transaction.createTransaction(randomAccount.address, 1, node.gAccount.password);
```

Comment:
Does `createTransaction()` params `address` and `password` represent a key pair? if yes it should be `randomAccount.password`